Gruntfile.js

```diff
@@ -55,7 +55,10 @@ module.exports = function (grunt) {
     'bower_components/webmaker-login-ux/dist/templates/ngWebmakerLogin.templates.js',
 
     'bower_components/imagesloaded/imagesloaded.pkgd.js',
-    'bower_components/masonry/dist/masonry.pkgd.js'
+    'bower_components/masonry/dist/masonry.pkgd.js',
+
+    'bower_components/angular-markdown-filter/markdown.js',
+    'bower_components/showdown/compressed/showdown.js'
```

Comment:
Just change this to Showdown. R+ with that, no need to put back on me for review.